### PR TITLE
Add index page

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**
Screenshot 2019-03-22 at 08 13 58
Screenshot 2019-03-22 at 08 14 07
Screenshot 2019-03-22 at 08 15 25
Screenshot 2019-03-22 at 08 16 58

### CODE REVIEW

File: source/theme/layout.html [hunk 1]
```diff
@@ -31,8 +31,12 @@
   <link rel="preconnect" href="https://centra.com" />-->
 </head>
 <body>
+  {% if pagename.startswith("index") %}
+  {% include 'landing-page.html' %}
```
Comment (paszkowskiDamian) [inline source/theme/layout.html]: this could be indented as well as body of else block.

Comment (NataliiaKononikhina) [reply source/theme/layout.html]: Done

File: source/theme/styles/components/_landing-page-hero-section.scss [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  max-width: 1265px;
+  margin: 0 auto;
+  letter-spacing: 0.12px;
+}
+
+.hero-section__content {
+  max-width: 560px;
+}
+
+.hero-section.white .hero-section__content{
```
Comment (paszkowskiDamian) [inline source/theme/styles/components/_landing-page-hero-section.scss]: This is scss you may use nested styles.

**Comment by paszkowskiDamian** [comment]
Looks very good even on IE 11, however one image is for some reason transparent

**Comment by paszkowskiDamian** [comment]
image

**Comment by kparzecki** [comment]
Screenshot 2019-03-25 at 15 07 56

We should use a non-breaking hyphen here (`&#x2011;`) or wrap `e-commerce` in a span with `white-space: nowrap` attribute.

File: source/theme/styles/components/_landing-page.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.landing-page__button .landing-page__button-arrow {
+  width: 20px;
+  height: 20px;
+  margin-left: 30px;
+}
+
+.landing-page__button.white .landing-page__button-arrow {
+  background: transparent url('./../../_static/img/vector-white.svg') no-repeat center center;
```
Comment (kparzecki) [inline source/theme/styles/components/_landing-page.scss]: Can we make it a pure css arrow?

Comment (NataliiaKononikhina) [reply source/theme/styles/components/_landing-page.scss]: Already done.

**Pull request merged**